docs/content/tutorial/pgbouncer.md

```diff
@@ -116,7 +116,7 @@ PGPASSWORD=randompassword psql -h localhost -p 5432 -U pgbouncer pgbouncer
 You should see something similar to this:
 
 ```
-psql (12.9, server 1.14.0/bouncer)
+psql (12.10, server 1.14.0/bouncer)
```

Comment:
❓  this is probably a nitpick but does the bouncer version need to be updated to 1.16?